Language/Functions/USB/Keyboard/keyboardModifiers.adoc

```diff
@@ -64,6 +64,18 @@ The Leonardo's definitions for modifier keys are listed below:
 |KEY_F10	|0xCB	|203
 |KEY_F11	|0xCC	|204
 |KEY_F12	|0xCD	|205
+|KEY_F13  |0xF0 |240
```

Comment:
The rest of the table uses true tabs for alignment. The formatting must be consistent through the table. I prefer to use spaces for alignment. Tabs are inappropriate for alignment due to the fact that their width depends on the preference of each reader.

So please either change your spaces to tabs, or else change the rest of the table to use spaces.